src/main/java/game/Game.java

```diff
@@ -205,12 +194,13 @@ public final void update(final List<String> input) {
 	 *            - the entity
 	 */
 	public final void checkCollision(final AbstractEntity e1) {
-		for (final AbstractEntity e2 : entities) {
-			if (!e1.equals(e2) && AbstractEntity.collision(e1, e2)
-					&& !destroyList.contains(e1) && !destroyList.contains(e2)) {
-				e1.collide(e2);
-			}
-		}
+		entities
```

Comment:
Ik begrijp niet hoe dit werkt, maar het lijkt te werken dus het is goed, maar kun je uitleggen wat je hier hebt gedaan?